src/components/download/DownloadSettings.js

```diff
@@ -205,7 +236,11 @@ const DownloadSettings = () => {
                                     : i18n.t('Close')}
                             </Button>
                             {isSupported && (
-                                <Button primary onClick={onDownload}>
+                                <Button
+                                    primary
+                                    disabled={!isRendered}
```

Comment:
Great! In puppeteer I can wait until the button is enabled and then take a screenshot.